src/main/java/io/github/moulberry/notenoughupdates/profileviewer/CrimsonIslePage.java

```diff
@@ -63,7 +63,7 @@ public class CrimsonIslePage extends GuiProfileViewerPage {
 		"Infernal"
 	};
 
-	private static final LinkedHashMap<String, String> apiDojoTestNames = new LinkedHashMap<String, String>() {{
+	public static final LinkedHashMap<String, String> apiDojoTestNames = new LinkedHashMap<String, String>() {{
```

Comment:
Imo these static methods should be moved into another class (hust repo hust) in the future.

Reply:
Imo this makes no sense at all to have these names in repo. They are not gonna change.